src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        Color randomColor = Color.values()[index];
+        return randomColor.name();
```

Comment:
Please, try to analize the result of the next code execution:
```
ColorSupplier supplier = new ColorSupplier();
for (int i = 0, i < 100; i++) {
    System.out.println(supplier. getRandomColor());
}
```
You should create a new random index for every `getRandomColor` method call.